content/en/docs/about/contributing.md

```diff
@@ -258,6 +258,7 @@ is the state of today.
     their [PR dashboard](https://k8s-gubernator.appspot.com/pr)
   - An **author** can work around this by manually reading the relevant OWNERS files,
     `/unassign`'ing unresponsive individuals, and `/assign`'ing others
+  - It is ultimately the PR creator's responsibility to see that the PR gets assigned to appropriate           **reviewers** and ping **reviewers** as needed
```

Comment:
This probably needs some additional context.

The PR creator should be responsible for escalating if need be and we should identify the appropriate escalation path

* e.g. ping the reviewers 
* If the reviewers don't respond look at the OWNERs file in root and ping approvers listed there
* If they don't respond email kubeflow-discuss

Reply:
Thanks @jlewi , updated accordingly.